src/singUp/components/albumList.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import { View, Text } from 'react-native';
+import axios from 'axios';
+
+
+
+
+class AlbumList extends Component {
+    state = {albums: [] };
```

Comment:
add spaces between brakets
```suggestion
    state = { albums: [] };
```